test/libsolidity/SemanticTest.cpp

```diff
@@ -47,6 +53,17 @@ SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVer
 	m_lineOffset(m_reader.lineNumber()),
 	m_enforceViaYul(enforceViaYul)
 {
+	using namespace placeholders;
+	auto simpleSmokeBuiltin = bind(&SemanticTest::builtinSmokeTest, this, _1);
```

Comment:
What is the `_1` here?

Reply:
Ah it is a bind feature. Btw `ContractABIUtils.cpp` uses `boost::bind`. Why not replace that too?